.gitignore

```diff
@@ -1,18 +1 @@
-# See http://help.github.com/ignore-files/ for more about ignoring files.
-#
-# If you find yourself ignoring temporary files generated by your text editor
-# or operating system, you probably want to add a global ignore instead:
-# git config --global core.excludesfile ~/.gitignore_global
-
-# Ignore bundler config
-/.bundle
-
-# Ignore the build directory
-/build
-
-# Ignore cache
-/.sass-cache
-/.cache
-
-# Ignore .DS_store file
-.DS_Store
```

Comment:
Hey, I fixed this mistake on another branch and pushed it already. Sorry about that.